web/pom.xml

```diff
@@ -28,6 +28,12 @@
       <groupId>org.kontinuity.catapult</groupId>
       <artifactId>catapult-core-impl</artifactId>
     </dependency>
+    <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-multipart-provider</artifactId>
+      <version>${version.org.jboss.resteasy_resteasy.multipart.provider}</version>
```

Comment:
yes, maybe a bug. but i don't want to go through the hoops and loops before we can leverage the swarm.jar here